spk/radarr/Makefile

```diff
@@ -35,6 +35,9 @@ WIZARDS_DIR = src/wizard/
 
 POST_STRIP_TARGET = radarr_extra_install
 
+# use alternate TMPDIR as /tmp might be too small.
+USE_ALTERNATE_TMPDIR = 1
```

Comment:
I think we should also add this to the other arr* packages, inc. Jackett https://github.com/SynoCommunity/spksrc/issues/4754